internal/scheduler/metrics/metrics.go

```diff
@@ -59,8 +60,9 @@ type Metrics struct {
 	matchedRegexIndexByErrorMessage *lru.Cache
 
 	// Job metrics.
-	transitions     *prometheus.CounterVec
-	resourceSeconds *prometheus.CounterVec
+	transitions                   *prometheus.CounterVec
+	resourceSeconds               *prometheus.CounterVec
+	unacknowledgedJobsPerExecutor *prometheus.GaugeVec
```

Comment:
I think we should do something more general. I'd call it
```
numUnacknoledgedJobsPerExecutor: prometheus.NewGaugeVec(
			prometheus.GaugeOpts{
				Namespace: namespace,
				Subsystem: subsystem,
				Name:      "unacknowledged_jobs_per_executor",
				Help:      "Number of unacknowledged jobs per executor.",
			},
			[]string{"executor"},
		),
```

Reply:
Separately, we may want to add another metric called something like `executorHealth`. But not now.